pkgs/development/interpreters/boron/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, fetchgit
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "boron";
+  version = "2.0.8";
+
+  src = fetchgit {
+    url = "https://git.code.sf.net/p/urlan/boron/code";
+    rev = "v${version}";
+    sha256 = "sha256-RpnMs9DqcbK/gLDH5x3/DvxodJb4tR76Yk8SbYtIkV4=";
+  };
+
+  # this is not a standard `configure`
+  configurePhase = ''
+    patchShebangs .
+    ./configure
+  '';
+
+  buildPhase = ''
+    export DESTDIR=$out
+    make -j
+    mkdir -p $out/include/boron
+    cp include/* $out/include/boron
+  '';
```

Comment:
This custom phase can be replaced with `makeFlags = [ "DESTDIR=$(out)" ];` + install target specification below.